pkgs/development/python-modules/colormath/default.nix

```
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, networkx
+, nose
+, numpy
+, lib
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "colormath";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "gtaylor";
+    rev = "3.0.0";
+    repo = "python-colormath";
+    sha256 = "1nqf5wy8ikx2g684khzvjc4iagkslmbsxxwilbv4jpaznr9lahdl";
+  };
+
+  # Are there no tests?
+  checkInputs = [ nose ];
+  checkPhase = "nosetests";
+
+  propagatedBuildInputs = [ networkx numpy ];
+
+  meta = {
+    description = "Color math and conversion library";
+    homepage = "https://github.com/gtaylor/python-colormath";
+    license = lib.licenses.bsd2;
+    maintainers = [ lib.maintainers.jonathanreeve ];
```

easier for other maintainers to join
```suggestion
    maintainers = with maintainers; [ jonathanreeve ];
```